serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -33,53 +26,59 @@ If using Maven, this means including following snippet in pom.xml:
 </dependency>
 ----
 
-== Configuring Smallrye channels
+The messaging capabilities are included in the Quarkus Serverless Workflow extension, even though the messaging capabilities are optional. This means you do not need to explicitly add the messaging add-on dependency when using Serverless Workflow. However, you need to add the Kafka Quarkus Smallrye connector dependency to indicate that you are using Apache Kafka.
```

Comment:
I think this change of order of the parragraph does not improve the reading experience. 
It is better to explain first why the dependency has to be added and then explain how to add the dependency, not the other way around

Reply:
The added dependency contains "messaging", therefore I thought of adding the dependency first and then discuss the messaging. 

However, to indicate why the dependency is added, we can change the order as follows:


You need to add the Kafka Quarkus Smallrye connector dependency to indicate that you are using Apache Kafka. To enable Serverless Workflow to use Apache Kafka Smallrye connector, add the following dependency to the `pom.xml` file of your project if using Maven:

.Add dependency for Apache Kafka Smallrye connector in `pom.xml`
[source,json]
----
<dependency>
  <groupId>io.quarkus</groupId>
  <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
</dependency>
----

The messaging capabilities are included in the Quarkus Serverless Workflow extension, even though the messaging capabilities are optional. This means you do not need to explicitly add the messaging add-on dependency when using Serverless Workflow.

Reply:
Ok, lets do that